src/p5.dance.js

```diff
@@ -199,6 +205,9 @@ module.exports = class DanceParty {
     this.analysisPosition_ = 0;
     this.playSound_({url: this.songMetadata_.file, callback: () => {this.songStartTime_ = new Date()}});
     this.p5_.loop();
+
+    /** Expose for testing **/
+    window.__mostRecentDancePartySongUrl = this.songMetadata_.file;
```

Comment:
Should we expose this on the code-dot-org side?

Reply:
It is certainly possible to only expose this only on the cdo side if we don't want access to it for unit tests on this side. I'm not sure that buys us a whole lot.